.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "prettier.trailingComma": "all",
+  "prettier.printWidth": 100,
+  "prettier.arrowParens": "avoid"
```

Comment:
If anyone prefers "always" happy for that instead!

Reply:
I'd be ok with leaving out the arrowParens always since those extra parens can sometimes add extra text which often isn't always shown in examples out there in the world. I'm happy to keep the 'avoid'